### PR TITLE
Improve class definition layout of new classes 

### PR DESCRIPTION
**Pull request by daschuer**
This adjust the layout of the recently introduced class definitions with the rest of Mixxx and our style guide.

### CODE REVIEW

File: src/widget/tooltipqopengl.cpp [hunk 1]
```diff
@@ -7,14 +7,18 @@
 #include <QToolTip>
 #include <memory>
 
-ToolTipQOpenGL::ToolTipQOpenGL() {
+#include "widget/wglwidget.h"
```
Comment (Swiftb0y) [inline src/widget/tooltipqopengl.cpp:10]: this is already included in the form of `<widget/wglwidget.h>` on line 3... I guess remove that one then?

File: src/widget/tooltipqopengl.cpp [hunk 2]
```diff
@@ -7,14 +7,18 @@
...
+
+ToolTipQOpenGL::ToolTipQOpenGL()
+        : m_active(true),
+          m_pWidget(nullptr) {
```
Comment (Swiftb0y) [inline src/widget/tooltipqopengl.cpp:14]: the tooltip is active without a widget? That doesn't seem right.

File: src/widget/tooltipqopengl.cpp (hunk 1 above)
Comment (daschuer) [reply src/widget/tooltipqopengl.cpp:10]: Removed.

File: src/widget/tooltipqopengl.cpp (hunk 2 above)
Comment (daschuer) [reply src/widget/tooltipqopengl.cpp:14]: The actual value does not matter because it is adjusted during MixxxMainWindow::initialize();

**Comment by daschuer** [comment]
Done.

File: src/widget/tooltipqopengl.cpp (hunk 2 above)
Comment (Swiftb0y) [reply src/widget/tooltipqopengl.cpp:12]: Ah I misinterpreted the meaning of the variable nevermind.

**Review by Swiftb0y — APPROVED** [review]
LGTM.

**Pull request merged**